### DISCUSSION TITLE
barbajs

### DISCUSSION THREAD

**Discussion by LfJohnVo** [livewire/livewire#4347] [1. Help]
Can i use barbaJS with livewire?, turbolinks its no longer manteined

**Comment by walirazzaq** [comment]
Not sure how well BarbaJs goes along livewire but Turbolinks is revamped now called [Turbo](https://turbo.hotwired.dev/reference/drive) and is being actively maintained.
Another option is [SwupJs](https://swup.js.org/plugins/livewire-plugin), I tried it and it works well
